lua/methods/mission.lua

```diff
@@ -5,17 +5,18 @@ local function exporter(object)
 
   local category = object:getCategory()
 
-  if(category == Object.Category.UNIT) then
+  if category == Object.Category.BASE or object.className_ == 'Airbase' then
+    -- carriers are of category unit, but are a Airbase class
+    return GRPC.exporters.airbase(object)
+  elseif category == Object.Category.UNIT then
     return GRPC.exporters.unit(object)
-  elseif(category == Object.Category.WEAPON) then
+  elseif category == Object.Category.WEAPON then
     return GRPC.exporters.weapon(object)
-  elseif(category == Object.Category.STATIC) then
+  elseif category == Object.Category.STATIC then
     return GRPC.exporters.static(object)
-  elseif(category == Object.Category.BASE) then
-    return GRPC.exporters.airbase(object)
-  elseif(category == Object.Category.SCENERY) then
+  elseif category == Object.Category.SCENERY then
     return GRPC.exporters.scenery(object)
-  elseif(category == Object.Category.Cargo) then
+  elseif category == Object.Category.Cargo then
```

Comment:
Removed the parenthesis for consistency with the other Lua code. But I'd also be fine to change everything to using parenthesis instead

Reply:
Parethessislessness is fine :grin: